odl/contrib/fom/test/test_supervised.py

```diff
@@ -124,6 +124,53 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_ssim(space):
```

Comment:
This is fine for now, but style wise (for next time), it could probably be written using fixtures for `force_lower_is_better` and `normalized`, which would significantly reduce duplication.